packages/flutter_nekoton_bridge/lib/nekoton/transport/jrpc_transport.dart

```diff
@@ -133,4 +133,8 @@ class JrpcTransport extends Transport {
 
   @override
   ArcTransportBoxTrait get transportBox => transport.innerTransport;
+
+  @override
+  String get connectionParamsHash =>
```

Comment:
Maybe we will call this method once during initialization?

Reply:
Yeah, for some reason it seemed to me that the endpoint[s] field was not final. Thank you.

Reply:
done!